data/json/recipes/food/brewing.json

```diff
@@ -0,0 +1,236 @@
+        "components": [
+            [ [ "water", 15 ], [ "water_clean", 15 ] ],
+            [ [ "cornmeal", 12 ], [ "buckwheat", 2 ] ],
+            [ [ "sugar", 100 ] ],
+            [ [ "yeast", 2 ] ]
+        ]
+    },{
+        "type" : "recipe",
+        "result": "brew_fruit_wine",
+        "result_mult" : 3,
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_DRINKS",
+        "skill_used": "cooking",
+        "difficulty": 3,
+        "time": 40000,
+        "batch_time_factors": [50, 4],
+        "autolearn": true,
+        "book_learn": [ ["brewing_cookbook", 1], ["family_cookbook", 2] ],
+        "qualities":[ { "id": "COOK", "level": 2 } ],
+        "tools": [ [ [ "hotplate", 6 ], [ "char_smoker", 6 ], [ "toolset", 6 ], [ "fire", -1 ] ] ],
+        "components": [
+            [ [ "water", 3 ], [ "water_clean", 3 ] ],
+            [
+                [ "pear", 2 ], [ "irradiated_pear", 2 ],
+                [ "apple", 2 ], [ "irradiated_apple", 2 ],
+                [ "peach", 2 ], [ "irradiated_peach", 2 ],
+                [ "plums", 2 ], [ "irradiated_plums", 2 ],
+                [ "apricot", 2 ], [ "irradiated_apricot", 2 ],
+                [ "juice", 1 ],
+                [ "blackberries", 2 ], [ "irradiated_blackberries", 2 ],
+                [ "blueberries", 2 ], [ "irradiated_blueberries", 2 ],
+                [ "cranberries", 6 ], [ "irradiated_cranberries", 6 ],
+                [ "raspberries", 2 ], [ "irradiated_raspberries", 2 ],
+                [ "strawberries", 2 ], [ "irradiated_strawberries", 2 ]
+            ]
+        ]
+    },{
+        "type" : "recipe",
+        "result": "brew_hb_beer",
+        "result_mult" : 15,
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_DRINKS",
+        "skill_used": "cooking",
+        "difficulty": 4,
+        "time": 80000,
+        "batch_time_factors": [50, 4],
+        "book_learn": [ ["brewing_cookbook", 4] ],
+        "qualities":[ { "id": "COOK", "level": 2 } ],
+        "tools": [ [ [ "hotplate", 6 ], [ "char_smoker", 6 ], [ "toolset", 6 ], [ "fire", -1 ] ] ],
+        "components": [
+            [ [ "water", 15 ], [ "water_clean", 15 ] ],
+            [ [ "barley", 3 ], [ "buckwheat", 3 ], [ "oats", 3 ] ],
+            [ [ "hops", 1 ] ],
+            [ [ "yeast", 1 ] ]
+        ]
+    },{
+        "type" : "recipe",
+        "result": "brew_mead",
+        "result_mult" : 3,
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_DRINKS",
+        "skill_used": "cooking",
+        "difficulty": 3,
+        "time": 80000,
+        "batch_time_factors": [50, 4],
+        "autolearn": true,
+        "book_learn": [ ["brewing_cookbook", 2] ],
+        "components": [
+            [ [ "water", 3 ], [ "water_clean", 3 ] ],
+            [ [ "honey_bottled", 10 ], [ "honeycomb", 2 ], [ "honey_glassed", 10 ] ],
+            [ [ "hops", 1 ] ],
```

Comment:
I don't think it commonly includes it. Also helps to differentiate it from the other recipes.